rzup/src/common.rs

```diff
@@ -0,0 +1,17 @@
+// Copyright 2024 RISC Zero, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+pub fn version() -> &'static str {
+    "0.2.0"
+}
```

Comment:
We should be able to grab the current version without needing to hardcode and then maintain this over time (since it's already specified in `Cargo.toml`). 

See: https://github.com/risc0/risc0/blob/main/risc0/zkvm/src/lib.rs#L155